pkg/gitreceive/build.go

```diff
@@ -237,6 +226,29 @@ func build(conf *Config, s3Client *storage.Client, kubeClient *client.Client, fs
 			return fmt.Errorf("Timed out waiting for object in storage, aborting build (%s)", err)
 		}
 	}
+
+	procType := pkg.ProcessType{}
+	if bType == buildTypeProcfile {
+		if _, err := os.Stat(fmt.Sprintf("%s/Procfile", tmpDir)); err == nil {
+			rawProcFile, err := ioutil.ReadFile(fmt.Sprintf("%s/Procfile", tmpDir))
+			if err != nil {
+				return fmt.Errorf("reading %s/Procfile", tmpDir)
+			}
+			if err := yaml.Unmarshal(rawProcFile, &procType); err != nil {
+				return fmt.Errorf("procfile %s/ProcFile is malformed (%s)", tmpDir, err)
+			}
+		} else {
+			log.Info("Procfile not present. Getting it from the buildpack")
```

Comment:
can you add a debug log under the info log, to indicate where we're getting the procfile from?